common/djangoapps/course_modes/models.py

```diff
@@ -26,6 +26,25 @@
                   ])
 
 
+class EnrollmentMode(models.Model):
```

Comment:
This is gone, but I added a unit test to check that settings.COURSE_ENROLLMENT_MODES has integers under 100.